pages/profile/components/asru-admin.jsx

```diff
@@ -33,16 +36,19 @@ class ASRUAdmin extends React.Component {
       return null;
     }
 
-    return <Fragment>
-      <hr />
-      <h2><Snippet>asru.title</Snippet></h2>
-      { this.renderRoles() }
-    </Fragment>
+    return (
+      <div className="panel-list">
+        <li>
```

Comment:
We shouldn't have `<li>` as a direct child of `<div>` - it should only ever appear inside a `<ul>` or `<ol>`

Reply:
Good spot, was just testing you were paying attention... :sweat_smile: